proposals/NNNN-atf-script-runner.md

```diff
@@ -0,0 +1,43 @@
+# ATF Script Runner
+
+* Proposal: [SDL-NNNN](NNNN-atf-script-runner.md)
+* Author: [Dmytro Boltovskyi](https://github.com/dboltovskyi)
+* Status: **Awaiting review**
+* Impacted Platforms: [ATF]
+
+## Introduction
+
+Originally ATF was developed for SDL developers and thats why it lacks a few features of modern Test Frameworks
+(Java jUnit, Python Nose, etc.)
+
+The purpose of this proposal is to develop extended script runner as part of ATF functionality.
+
+## Motivation
+
+Currently ATF is unable to:
+
+  - run multiple test scripts
+  - run test script in parallel
+  - create test reports in standard format
+
+## Proposed solution
+
+Develop script runner with the following features:
+
+  - Ability to run single test script
```

Comment:
@dboltovskyi 
I think 'option' is more appropriate in that context than 'ability'